.github/workflows/prestocpp-worker-with-clp-connector-runtime-image-build.yml

```diff
@@ -0,0 +1,97 @@
+name: prestocpp-worker-with-clp-connector-runtime-image-build
+
+on:
+  # TODO: specifiy the branch to the release-0.293 when finalize the PR
+  pull_request:
+  push:
+
+jobs:
+  prestocpp-worker-with-clp-connector-runtime-image:
+    name: prestocpp-worker-with-clp-connector-runtime-image
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: "actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683"
+        with:
+          submodules: "recursive"
+
+      - name: "Login to Image Registry"
+        uses: "docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772"
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+
+      - name: "Update Metadata for Dependency Image"
+        id: "meta-dependency"
+        uses: "docker/metadata-action@902fa8ec7d6ecbf8d84d538b9b233a880e428804"
+        with:
+          images: >-
+            ghcr.io/${{github.repository}}/prestissimo-with-clp-connector-dependency
+          tags: |
+            type=raw,value=ubuntu-22.04
+
+      - name: "Update Metadata for Runtime Image"
+        id: "meta-runtime"
+        uses: "docker/metadata-action@902fa8ec7d6ecbf8d84d538b9b233a880e428804"
+        with:
+          images: >-
+            ghcr.io/${{github.repository}}/prestissimo-with-clp-connector-runtime
+          tags: |
+            type=raw,value=ubuntu-22.04
+
+      - name: Check for dependency changes or missing dependency image
+        id: dependency-changes
+        run: |
+          git fetch origin ${{ github.event.before }} --depth=1 || true
+          dep_changed=false
+          if git diff --name-only ${{ github.event.before }} ${{ github.sha }} | grep -q '^presto-native-execution/scripts'; then
+            dep_changed=true
+          fi
+
+          velox_old_sha=$(git ls-tree ${{ github.event.before }} presto-native-execution/velox | awk '{print $3}')
+          velox_new_sha=$(git ls-tree ${{ github.sha }} presto-native-execution/velox | awk '{print $3}')
+          cd presto-native-execution/velox
+          if git diff --name-only ${velox_old_sha} ${velox_new_sha} | grep -q '^scripts'; then
+            dep_changed=true
+          fi
+          echo "dep-changed=${dep_changed}" >> $GITHUB_OUTPUT
+
+          image_exists=true
+          if ! docker manifest inspect "${{ steps.meta-dependency.outputs.tags }}" > /dev/null 2>&1; then
+            image_exists=false
+          fi
+          echo "image-exists=${image_exists}" >> $GITHUB_OUTPUT
```

Comment:
I think there's a better way to do this using dorny/paths-filter that we use in clp. So for now, can we delete this and defer the effort to another PR?